app/ai-gateway/ai-audit-log-reference.md

```diff
@@ -40,117 +40,270 @@ You can also use [logging plugins](/plugins/?category=logging) to route these lo
 
-## Cache logging {% new_in 3.8 %}
+If you're using the [AI RAG Injector plugin](/plugins/ai-rag-injector/), AI Gateway logs include additional fields that provide detailed information about the retrieval-augmented generation process. These fields track the vector database used, whether relevant context was injected into the prompt, the latency of data fetching, and embedding metadata such as tokens used and the provider/model details.
+
+The following fields appear in structured AI logs when the AI RAG Injector plugin is enabled:
+
+{% table %}
+columns:
+  - title: Property
+    key: property
+  - title: Description
+    key: description
+rows:
+  - property: "`ai.proxy.rag-inject.vector_db`"
+    description: The vector database used (for example, `pgvector`).
+  - property: "`ai.proxy.rag-inject.injected`"
+    description: Boolean indicating if RAG injection occurred.
+  - property: "`ai.proxy.rag-inject.fetch_latency`"
+    description: The fetch latency in milliseconds.
+  - property: "`ai.proxy.rag-inject.chunk_ids`"
+    description: List of chunk IDs retrieved.
+  - property: "`ai.proxy.rag-inject.embeddings_latency`"
+    description: Time taken to generate embeddings, in milliseconds.
+  - property: "`ai.proxy.rag-inject.embeddings_tokens`"
+    description: Number of tokens used for embeddings.
+  - property: "`ai.proxy.rag-inject.embeddings_provider`"
+    description: Provider used to generate embeddings.
+  - property: "`ai.proxy.rag-inject.embeddings_model`"
+    description: Model used to generate embeddings.
+{% endtable %}
+
+### AI Semantic Cache logs {% new_in 3.8 %}
 
 If you're using the [AI Semantic Cache plugin](/plugins/ai-semantic-cache), AI Gateway logs include additional fields under the cache object for each plugin entry. These fields provide insight into cache behavior—such as whether a response was served from cache, how long it took to fetch, and which embedding provider and model were used if applicable.
 
-The following example shows how cache-related metadata appears alongside usage and model details in a structured AI log entry:
+The following fields appear in AI logs when semantic caching is enabled:
+
+{% table %}
+columns:
+- title: Property
+  key: property
+- title: Description
+  key: description
+rows:
+- property: "`ai.proxy.cache.cache_status`"
+  description: |
+    {% new_in 3.8 %} The cache status. This can be `Hit`, `Miss`, `Bypass`, or `Refresh`.
+- property: "`ai.proxy.cache.fetch_latency`"
+  description: |
+    The time, in milliseconds, it took to return a cache response.
+- property: "`ai.proxy.cache.embeddings_provider`"
+  description: |
+    For semantic caching, the provider used to generate the embeddings.
+- property: "`ai.proxy.cache.embeddings_model`"
+  description: |
+    For semantic caching, the model used to generate the embeddings.
+- property: "`ai.proxy.cache.embeddings_latency`"
+  description: |
+    For semantic caching, the time taken to generate the embeddings.
```

Comment:
```suggestion
    The provider used to generate the embeddings.
- property: "`ai.proxy.cache.embeddings_model`"
  description: |
    The model used to generate the embeddings.
- property: "`ai.proxy.cache.embeddings_latency`"
  description: |
    The time taken to generate the embeddings.
```
Since you split these by plugin, I figured that bit wasn't needed